cvat-ui/src/actions/annotation-actions.ts

```diff
@@ -295,28 +295,59 @@ async function fetchAnnotations(frameNumber?: number): Promise<{
     };
 }
 
+function navigationInterruptionDetected(callback: (interrupted: boolean) => void): void {
+    let currentPath = window.location.pathname;
+
+    // Listen for popstate events (back/forward navigation)
+    window.addEventListener('popstate', () => {
+        const newPath = window.location.pathname;
+        if (newPath !== currentPath) {
+            console.log('Navigation interruption detected.');
```

Comment:
we do not need logging in production code

Reply:
Let me remove all